data/sources/admin-boundaries.json

```diff
@@ -42,6 +42,16 @@
       "sql": "SELECT the_geom_webmercator, puma, puma AS id, puma AS geoid FROM dcp_puma",
       "dataPipeline": true
     },
+    {
+      "id": "cdtas",
+      "sql": "SELECT the_geom_webmercator, cdtaname FROM nycdta2020",
+      "dataPipeline": true
```

Comment:
I don't think these are part of the GIS data pipeline? These are from newly uploaded shapefiles right? You should  set these to `false` unless i'm mistaken. 
Also by setting `dataPipeline` to `true`, it indicates that this data source is in both our production (labs_dl) and staging (plannignweb) carto

Reply:
Updated.

We have a staging carto?  You learn something new every day.